src/DustExtinction.jl

```diff
@@ -16,6 +18,8 @@ include("dust_maps.jl")
 
 @deprecate ccm89(x::AbstractArray, r_v::Real = 3.1) ccm89.(x, r_v)
 @deprecate od94(x::AbstractArray, r_v::Real = 3.1) od94.(x, r_v)
+@deprecate vcg04(x::AbstractArray, r_v::Real = 3.1) vcg04.(x, r_v)
+@deprecate gcc09(x::AbstractArray, r_v::Real = 3.1) gcc09.(x, r_v)
```

Comment:
I wouldn't say you need to add these deprecation statements (notice `cal00` isn't there).

Reply:
Done!